src/preact/index.js

```diff
@@ -106,3 +106,13 @@ export function useLayoutEffect(effect, opt_deps) {
 export function useContext(context) {
   return hooks.useContext(context);
 }
+
+/**
+ * @param {function():(T|undefined)} cb
```

Comment:
I think the cb is supposed to return `undefined` in all cases, right? Does React ever use the return value?

Reply:
I thought a return value was possible but uncommon, but it looks like here it's expected to be `undefined`: https://github.com/preactjs/preact/blob/master/hooks/src/index.js#L212-L218

Reply:
Hmm. I'm seeing return values used very commonly for `useCallback`.

Reply:
Ok, I think I'm wrong (and preact). The React types allow any return value: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f0f57b7b5a871f801c7f3f81490897b7cb7d765d/types/react/index.d.ts#L1104